<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>322</width>
      <height>168</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="pickDiv">
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>c</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>c</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>b</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="eventName">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>200</number>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="year">
         <property name="minimum">
          <number>1984</number>
         </property>
         <property name="maximum">
          <number>2025</number>
         </property>
         <property name="value">
          <number>2020</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randtozipbutton">
         <property name="text">
          <string>1 - randtozip</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="afterZipButton">
         <property name="toolTip">
          <string>tooltip</string>
         </property>
         <property name="text">
          <string>2 - after zip organize</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="spreadsheetButton">
         <property name="text">
          <string>3 - spreadsheet upload</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>randtozipbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>randtozip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>afterZipButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>beyondzip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spreadsheetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>spreadsheetup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1(bool)</signal>
  <slot>slot1()</slot>
  <slot>randtozip()</slot>
  <slot>pressEnter()</slot>
  <slot>beyondzip()</slot>
  <slot>spreadsheetup()</slot>
 </slots>
</ui>
